subsys/mgmt/ec_host_cmd/backends/ec_host_cmd_backend_spi_stm32.c

```diff
@@ -536,13 +536,28 @@ static int prepare_rx(struct ec_host_cmd_spi_ctx *hc_spi)
 	int ret;
 
 	hc_spi->prepare_rx_later = 0;
+
+#ifdef EC_HOST_CMD_ST_STM32H7
```

Comment:
Note: `st,stm32h7-spi` is also used in other series like H5, U5, WBA.
I didn't check the RM for those, but you might want to check that tis behavior is related to the IP rather than the series.

Reply:
Thanks for the info. I've checked and HAL provides used functions inside `#ifdef EC_HOST_CMD_ST_STM32H7` also for H5, U5 and WBA chips.